tests/core/data_structures/test_result.py

```
@@ -151,7 +153,7 @@ def test_result_is_pickleable() -> None:
     result = Simulator(parallel=False, show_progress=False).run(state, H, sim_params)
 
     blob = pickle.dumps(result)
-    restored = pickle.loads(blob)  # ruff:ignore[suspicious-pickle-usage]  # test-only: bytes are produced one line above (round-trip)
+    restored = pickle.loads(blob)  # ruff: ignore[suspicious-pickle-usage]  # controlled round-trip of bytes produced above
```

Ruff (0.16.1)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)